publication-commons/src/main/java/no/unit/nva/publication/model/business/UserInstance.java

```diff
@@ -108,7 +108,8 @@ public static UserInstance fromMessage(Message message) {
     }
 
     public static UserInstance fromTicket(TicketEntry ticket) {
-        return UserInstance.create(ticket.getOwner(), ticket.getCustomerId());
+        return new UserInstance(ticket.getOwner().toString(), ticket.getCustomerId(), ticket.getOwnerAffiliation(),
+                                ticket.getResponsibilityArea(), null, List.of(), UserClientType.INTERNAL);
```

Comment:
UserInstance created from ticket is missing important fields such as owner affiliation and responsibility area.